### ISSUE TITLE
Upgrade of private messages

### ISSUE THREAD

**Issue by inzerat** [buddycore/surface-theme#47]
1) If is text of private message too long, do read more
2) Seen date - please do date with time (date & time reading)
3) Why is there SSS? Please change text on "Mark as read" and "Mark as unread" It confuses people
4) When someone wants to see the messages in the conversation, please do that he will automatically jump down the page on the last message. Look how it works on Facebook, it also jumps down and you do not have to scroll down in order to write the answer.

![untitled](https://cloud.githubusercontent.com/assets/21200726/19636416/84a7f4de-99c9-11e6-94f9-fc0b19cd07c1.png)

**Comment by buddycore** [comment]
1) Does not make sense in a message thread, when browsing all messages it's only an excerpt.
2) This is buggy, I've had a problem implementing. Let me try again.
3) I will fix this.
4) When clicking the link in the header I will make sure to link this to the most recent message.

Thanks for your feedback, this will help make the theme a better theme.

Unfortunately for now my second monitor is broken so I will try to make these changes asap.

**Comment by inzerat** [comment]
4) Link in the header? I thought Link to detail the conversation.

**Comment by inzerat** [comment]
5 ) Load more button - In some of the conversations I have more than 30 messages, then it's confusing. New message button - Could you indicate a new incoming message as in the activities?
6 ) New messages at the top, old messages down. Similar as is the case activity. Reply form at the top as well.

![untitled](https://cloud.githubusercontent.com/assets/21200726/19961778/aef41fc2-a1b4-11e6-9279-609148eb297f.png)

**Comment by inzerat** [comment]
Tutorial https://buddypress.org/support/topic/sorting-private-message-threads/

**Comment by inzerat** [comment]
New messages button - may be confused, better will be text: "Show new messages in this conversation"

**Comment by buddycore** [comment]
I can reverse the order now with that helpful tutorial.

Everything else is too much for this theme and I wouldn't know how to build those things at the moment.

**Comment by buddycore** [comment]
I've implemented this but have been unable to add a load more button and also add new message replies via ajax.

If you wish further modifications you should seek out a developer on buddypress.org
